<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>780</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1040</width>
    <height>780</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="tabBarAutoHide">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionSaveAll"/>
    <addaction name="actionClose"/>
    <addaction name="actionCloseAll"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionPrintView"/>
    <addaction name="actionPDFExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionSelectAll"/>
    <addaction name="separator"/>
    <addaction name="actionComment"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>风格(&amp;S)</string>
    </property>
    <addaction name="actionTheme"/>
    <addaction name="actionBackground"/>
    <addaction name="separator"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionInstruction"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>文档(&amp;V)</string>
    </property>
    <addaction name="actionSearch"/>
    <addaction name="actionJumpRow"/>
    <addaction name="actionOpenGLAPI"/>
    <addaction name="separator"/>
    <addaction name="actionPrevious"/>
    <addaction name="actionNext"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_File"/>
   <addaction name="menu_2"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionSelectAll"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="separator"/>
   <addaction name="actionSearch"/>
   <addaction name="actionOpenGLAPI"/>
   <addaction name="actionJumpRow"/>
   <addaction name="actionComment"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/new.png</normaloff>:/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>新建文件(&amp;N)...</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件(&amp;O)...</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/saveAs.png</normaloff>:/icons/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)...</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionSaveAll">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/saveall.png</normaloff>:/icons/saveall.png</iconset>
   </property>
   <property name="text">
    <string>全部保存(&amp;G)</string>
   </property>
   <property name="toolTip">
    <string>全部保存</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/close.png</normaloff>:/icons/close.png</iconset>
   </property>
   <property name="text">
    <string>关闭(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>关闭</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/closeAll.png</normaloff>:/icons/closeAll.png</iconset>
   </property>
   <property name="text">
    <string>全部关闭(&amp;H)</string>
   </property>
   <property name="toolTip">
    <string>全部关闭</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/print.png</normaloff>:/icons/print.png</iconset>
   </property>
   <property name="text">
    <string>打印(&amp;P)...</string>
   </property>
   <property name="toolTip">
    <string>打印</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/undo.png</normaloff>:/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
   <property name="toolTip">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/redo.png</normaloff>:/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>恢复(&amp;R)</string>
   </property>
   <property name="toolTip">
    <string>恢复</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/cut.png</normaloff>:/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/copy.png</normaloff>:/icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;P)</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/paste.png</normaloff>:/icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/selectall.png</normaloff>:/icons/selectall.png</iconset>
   </property>
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>全选</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionComment">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/comment.png</normaloff>:/icons/comment.png</iconset>
   </property>
   <property name="text">
    <string>区块注释/取消注释(&amp;Z)...</string>
   </property>
   <property name="toolTip">
    <string>区块注释/取消注释</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+/</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
   </property>
   <property name="text">
    <string>查找(&amp;F)...</string>
   </property>
   <property name="toolTip">
    <string>查找</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionTheme">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/theme.png</normaloff>:/icons/theme.png</iconset>
   </property>
   <property name="text">
    <string>主题(&amp;T)...</string>
   </property>
   <property name="toolTip">
    <string>主题</string>
   </property>
  </action>
  <action name="actionBackground">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/background.png</normaloff>:/icons/background.png</iconset>
   </property>
   <property name="text">
    <string>背景图片(&amp;B)</string>
   </property>
   <property name="toolTip">
    <string>背景图片</string>
   </property>
  </action>
  <action name="actionInstruction">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/instruction.png</normaloff>:/icons/instruction.png</iconset>
   </property>
   <property name="text">
    <string>使用说明(&amp;I)</string>
   </property>
   <property name="toolTip">
    <string>使用说明</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>关于Shaderpad++(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>关于Shaderpad++</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Qt.png</normaloff>:/icons/Qt.png</iconset>
   </property>
   <property name="text">
    <string>关于Qt(&amp;Q)</string>
   </property>
   <property name="toolTip">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/next.png</normaloff>:/icons/next.png</iconset>
   </property>
   <property name="text">
    <string>下一个文档(&amp;N)</string>
   </property>
   <property name="toolTip">
    <string>下一个文档</string>
   </property>
   <property name="shortcut">
    <string>PgDown</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/previous.png</normaloff>:/icons/previous.png</iconset>
   </property>
   <property name="text">
    <string>前一个文档(&amp;P)</string>
   </property>
   <property name="toolTip">
    <string>前一个文档</string>
   </property>
   <property name="shortcut">
    <string>PgUp</string>
   </property>
  </action>
  <action name="actionPrintView">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/view.png</normaloff>:/icons/view.png</iconset>
   </property>
   <property name="text">
    <string>打印预览(&amp;V)</string>
   </property>
   <property name="toolTip">
    <string>打印预览</string>
   </property>
  </action>
  <action name="actionPDFExport">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/pdf.png</normaloff>:/icons/pdf.png</iconset>
   </property>
   <property name="text">
    <string>导出PDF文件(&amp;E)</string>
   </property>
   <property name="toolTip">
    <string>导出PDF文件</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/ZoomIn.png</normaloff>:/icons/ZoomIn.png</iconset>
   </property>
   <property name="text">
    <string>增大字体(&amp;U)
</string>
   </property>
   <property name="toolTip">
    <string>增大字体</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/ZoomOut.png</normaloff>:/icons/ZoomOut.png</iconset>
   </property>
   <property name="text">
    <string>减小字体(&amp;D)
</string>
   </property>
   <property name="toolTip">
    <string>减小字体</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionOpenGLAPI">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/glsl.png</normaloff>:/icons/glsl.png</iconset>
   </property>
   <property name="text">
    <string>OpenGL着色器API文档查询(&amp;L)</string>
   </property>
   <property name="toolTip">
    <string>着色器API文档查询</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionJumpRow">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Jump.png</normaloff>:/icons/Jump.png</iconset>
   </property>
   <property name="text">
    <string>行跳转(&amp;T)</string>
   </property>
   <property name="toolTip">
    <string>行跳转</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
